app/main.development.js

```diff
@@ -175,11 +179,80 @@ const installExtensions = async () => {
   }
 };
 
+const {ipcMain} = require("electron");
+
+ipcMain.on("start-daemon", (event, arg) => {
+  if (dcrdPID) {
+    logger.log("info", "dcrd already started " + dcrwPID);
+    event.returnValue = dcrdPID;
+    return;
+  }
+  logger.log("info", "launching dcrd with " + arg);
```

Comment:
The output for this is: `info: launching dcrd with [object Object]` so we should maybe use `JSON.stringify()` here.